### PR TITLE
Switch to pixi

### PR DESCRIPTION
**Pull request by znichollscr**
Based on the discussion in https://github.com/IAMconsortium/concordia/issues/66, we need to install dependencies with conda. The best tool for this (in my experience) is pixi. It's basically conda, except the lock files work.

cc @chrisroadmap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,14 +133,16 @@ In this repository, we use the following tools:
...
-  (for more on environment management, see
-  [general principles: environment management](https://gitlab.com/znicholls/mullet-rse/-/blob/main/book/theory/environment-management.md))
-    - there are lots of environment management systems. Poetry works and for
-      simple projects like this there is no need to overcomplicate things
-    - we track the `poetry.lock` file so that the environment is completely
-      reproducible on other machines or by other people (e.g. if you want a
-      colleague to take a look at what you've done)
+- [Pixi](https://pixi.sh/latest/) for environment management
+   (for more on environment management, see
+   [general principles: environment management](https://gitlab.com/znicholls/mullet-rse/-/blob/main/book/theory/environment-management.md))
```
Comment (jkikstra) [inline README.md:138]: Cannot open this link

Comment (znicholls) [reply README.md:138]: My bad, try again

**Review by jkikstra — APPROVED** [review]
I get some error with `import aneris` (`No module named 'pycountry'`) on this one, but since that's solved (I checked) in #13 (probably in #12) I don't mind it here.

**Comment by znicholls** [comment]
> I get some error with `import aneris` (`No module named 'pycountry'`) on this one, but since that's solved (I checked) in #13 (probably in #12) I don't mind it here.

Yep I couldn't work get everything running without touching lots of things so you probably won't be able to actually run things until #11 or #12

**Comment by jkikstra** [comment]
Yeah, great. I'm testing on #13 now (basically #12 with extra start for BB4CMIP), and will just read the changes of each PR without testing each separately, and then merge everything up until (and including) #12.

**Pull request merged**